package/MDAnalysis/coordinates/TRJ.py

```diff
@@ -768,18 +809,24 @@ class NCDFWriter(base.WriterBase):
         name of output file
     n_atoms : int
         number of atoms in trajectory file
-    start : int (optional)
```

Comment:
https://docs.mdanalysis.org/2.0.0-dev0/documentation_pages/coordinates/init.html#trajectory-writer-class seems to say that start/stop/step should be available. Not sure if this still makes sense (you know, after >5 years when the trajectory API was probably updated last time).

WriterBase https://github.com/MDAnalysis/mdanalysis/blob/55ffd255013dac427be3a8f4cf7962a4a886b712/package/MDAnalysis/coordinates/base.py#L2204 is not clear about what it requires, the API docs are really what counts.

This would be a good time to clean up the trajectory API, too, at least the Writer part.

Reply:
So looking at the other main writers (DCD, XDR, LAMMPS), it doesn't seem like there is anything that uses start/step/delta (or even has it as a parameter for `_init_`). The closest we have is `step` and `dt` for the DCDWriter, although worth nothing that it's not immediately clear to me that `step` actually does anything!

It's possible I'm missing something, but since it doesn't look like other writers don't also meet the API's requirements to have these (either as parameters to init or as class attributes), I'd suggest we just clean up the API docs in a separate PR instead?

Reply:
yes, separate PR

Reply:
See #3392 3392